WordPressAuthenticator/Analytics/AnalyticsTracker.swift

```diff
@@ -215,9 +215,13 @@ public class AnalyticsTracker {
         case createAccount = "create_account"
     }
     
-    /// Provides the sign-in tracking state machine that can be shared across different trackers if needed.
+    /// Shared Instance.
     ///
-    public class Context {
+    public static var shared = AnalyticsTracker()
+    
+    /// State for the analytics tracker.
+    ///
+    private class State {
```

Comment:
Since this won't be shared or public anymore, it now represents an internal state.